documentation/modules/ref-brokered-infra-config-override-probe-timeout.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// assembly-infrastructure-configuration.adoc
+
+[id='ref-brokered-infra-config-override-probe-timeout-{context}']
+= Overriding the readiness probe timing for brokered infrastructure configuration
+
+You can override the default value for the readiness probe timing
+
+The following example shows 
+
```

Comment:
How about.. "The following example shows how to override some default values of the readiness probe timeout for the broker resource."